rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1681,6 +1682,11 @@ def get_hashchainlog(entry):
 
     assert cursor.execute('SELECT MAX(seq) FROM location').fetchone()[0] == 40
 
+    # get settings and get that saddle is present
+    cursor.execute('SELECT value FROM settings WHERE name="current_price_oracles"')
+    assert 'saddle' in cursor.fetchone()[0]
```

Comment:
Can you instead save the value of the old oracles list? And then later in the after-upgrade check, test that the list is same but excludes saddle?